search-app/resources/templates/search-provider-tag-landing-links.html

```diff
@@ -1,14 +1,39 @@
 {% extends "templates/search-base.html" %}
 
-{% block title %}{{ app-title }} - Landing Pages for the {{ provider-name}} {{ tag-name }} Collections{% endblock %}
+{% block title %}{{ app-title }} - Landing Pages for the {{ provider-id}} {{ tag-name }} Collections{% endblock %}
```

Comment:
Removed "the" in the title.

Note that `provider-id` and `short-name` in the data structure are identical for all providers; in the part of the code that renders this page, we get `provider-id` for free: it's passed and parsed from URL in the request. To get the `short-name`, we'd have to issue another query ... and due to that expense and the fact that they are identical, we're just going with the id for now. 

Fortunately, this is not exposed to the user -- all they will see is "ASF" or "NSIDC", etc. ... unless I missed something, and we're making a reference to the id when we shouldn't be ...